cypress/e2e/updateUser.cy.ts

```diff
@@ -76,9 +76,9 @@ context('Coordinape', () => {
       });
 
     // enter the fixed payment amount
-    cy.getInputByLabel('Fixed Payment Amount').clear().type('1200').blur();
+    cy.getInputByLabel('Fixed Payment Amount').clear().type('12000').blur();
```

Comment:
The test is failing here on this 12000 value, i guess the 0 is still appended to the end of the number.
```suggestion
    cy.getInputByLabel('Fixed Payment Amount').clear().type('1200').blur();
```

@wingfeatherwave

Reply:
i was actually just thinking of removing this test. i don't think it's high-value enough to incur the time cost of Cypress, since the first test in this file already verifies that the form is working.